deploy/a8s/manifests/service-binding-controller.yaml

```diff
@@ -331,24 +397,29 @@ spec:
       app.kubernetes.io/component: controller-manager
       app.kubernetes.io/name: service-binding-controller-manager
       app.kubernetes.io/part-of: a8s-service-binding
+      control-plane: controller-manager
   template:
     metadata:
       annotations:
+        kubectl.kubernetes.io/default-container: manager
         prometheus.io/port: "8443"
         prometheus.io/scrape: "true"
       labels:
         app.kubernetes.io/component: controller-manager
         app.kubernetes.io/name: service-binding-controller-manager
         app.kubernetes.io/part-of: a8s-service-binding
+        control-plane: controller-manager
     spec:
       containers:
       - args:
-        - --config=/config/controller_manager_config.yaml
```

Comment:
Why remove this when the config map is still mounted in line 449?

Reply:
Is it an option to remove the `volumeMount` then?

Reply:
We will handle bringing back this configurability to the service binding controller as a separate work item.

Reply:
Will be solved in [A8S-2387](https://anynines.atlassian.net/browse/A8S-2387)